### DISCUSSION TITLE
gallery页面

### DISCUSSION THREAD

**Discussion by missdeng-fu** [MaLuns/hexo-theme-async#112] [Q&A 疑难杂症]
我的gallery页面总是不成功，页面总是空白，而且每次hexo g -d的时候，都会出错，但是我已经换了最新的page.jes了
![Uploading {5DB72C28-F407-471f-AD7B-742ED4850CF9}.png…]()

**Comment by missdeng-fu** [comment]
<img width="734" alt="{5DB72C28-F407-471f-AD7B-742ED4850CF9}" src="https://github.com/MaLuns/hexo-theme-async/assets/66002735/63a990ab-aa51-473a-a039-2b36f18c1d18">

**Comment by MaLuns** [comment]
gallery页面本质是 hexo 的自定义页面，看看是不是有地方语法格式写错了

**Comment by missdeng-fu** [comment]
可是我后来从您的那个github上专门把这个文件原封不动的复制过来也不行



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年1月2日(周二) 中午12:05
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [MaLuns/hexo-theme-async] gallery页面 (Discussion #112)




 
gallery页面本质是 hexo 的自定义页面，看看是不是有地方语法格式写错了

**Comment by MaLuns** [comment]
https://github.com/MaLuns/hexo-theme-async/tree/dev/demo/source/gallery  你参考这个看看

**Comment by missdeng-fu** [comment]
我看过这个了，目前我的问题是，无法创建新的类似gallery的新页



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年1月2日(周二) 中午12:20
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [MaLuns/hexo-theme-async] gallery页面 (Discussion #112)




 
https://github.com/MaLuns/hexo-theme-async/tree/dev/demo/source/gallery  你参考这个看看
